src-plugins/itkFilters/itkFiltersToolBox.cpp

```diff
@@ -106,13 +101,6 @@ itkFiltersToolBox::itkFiltersToolBox ( QWidget *parent ) : medFilteringAbstractT
 
     QObject::connect ( d->filters, SIGNAL ( currentIndexChanged ( int ) ), this, SLOT ( onFiltersActivated ( int ) ) );
 
-    QLabel * dataTypeLabel = new QLabel ( tr ( "Data type :" ) );
```

Comment:
Haven't thought about this really. It's not really a priority, so we can discuss this at a meeting someday.